mesheryctl/internal/cli/root/system/start.go

```diff
@@ -108,32 +115,42 @@ func start() error {
 		if err != nil {
 			return err
 		}
+
 		services := compose.Services // Current Services
+		//extracting the custom user port from config.yaml
+		userPort := strings.Split(currCtx.Endpoint, ":")
+		userPortMapping := userPort[len(userPort)-1] + ":8080"
```

Comment:
The hardcoded port 8080 here is a dangerous assumption.

@kumarabd have you updated the helm chart to use 9081, yet?

Reply:
Agreed, @leecalcote
I have updated it to use the container port mentioned in the `meshery.yaml` port mapping for `meshery`. 
But as long as `system start` downloads `meshery.yaml` on start everytime, it will remain 8080. That's why I had hardcoded it earlier.

Reply:
@lee yes the helm chart is updated for 9081.